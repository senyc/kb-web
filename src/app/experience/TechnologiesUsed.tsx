interface TechnologiesUsedProps {
  items: Array<string>;
}

export default function TechnologiesUsed({ items }: TechnologiesUsedProps) {
  return (
    <div className='grid sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5'>
      {items.map((item, idx) => (
        <li className='list-disc' key={idx}>
          {item}
        </li>
      ))}
    </div>
  );
}
