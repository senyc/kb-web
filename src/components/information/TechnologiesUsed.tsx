interface TechnologiesUsedProps {
  items: Array<string>;
}

export default function TechnologiesUsed({ items }: TechnologiesUsedProps) {
  return (
    <ul className='ml-6 grid  grid-cols-1 sm:grid-cols-2 md:grid-cols-4 lg:grid-cols-5'>
      {items.map((item, idx) => (
        <li className='list-disc' key={idx}>
          {item}
        </li>
      ))}
    </ul>
  );
}
