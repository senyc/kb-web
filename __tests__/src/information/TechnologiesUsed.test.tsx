import { create } from 'react-test-renderer';

import TechnologiesUsed from "@components/information/TechnologiesUsed";

describe('TechnologiesUsed elements match recorded snapshots', () => {
  describe('TechnologiesUsed', () => {
    it('renders items', () => {
      const tree = create(<TechnologiesUsed items={['one', 'two', 'three']} />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
