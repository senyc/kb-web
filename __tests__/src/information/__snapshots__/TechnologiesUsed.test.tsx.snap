// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnologiesUsed elements match recorded snapshots TechnologiesUsed renders items 1`] = `
<ul
  className="ml-6 grid  grid-cols-1 sm:grid-cols-2 md:grid-cols-4 lg:grid-cols-5"
>
  <li
    className="list-disc"
  >
    one
  </li>
  <li
    className="list-disc"
  >
    two
  </li>
  <li
    className="list-disc"
  >
    three
  </li>
</ul>
`;
